drives.go

```diff
@@ -0,0 +1,63 @@
+package firecracker
+
+import (
+	"strconv"
+
+	models "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
+)
+
+// DrivesBuilder is a builder that will build an array of drives used to set up
+// the firecracker microVM. The DriveID will be an incrementing number starting
```

Comment:
(I'm not familiar enough with the Firecracker API yet) What are the requirements around the DriveID? Is it just uniqueness or are there other requirements too?

Reply:
Have a look on conversation [here](https://github.com/firecracker-microvm/firecracker-containerd/pull/49). It adds some context to this changes.

Reply:
Based on the docs it'll stomp over any pre-existing drive with the same ID, and no requirements I see other than `DriveID` not being empty which is returned from firecracker's API.

Reply:
@mxpv I saw that, but that didn't answer my question.  @xibz's response is what I was looking for.